js/services/SettingsManager.js

```diff
@@ -33,12 +33,17 @@ define([
 
         getPublicSSHKey: function () {
             return new Promise(function (resolve, reject) {
-
-                FS.readFile(PUBLIC_KEY_PATH).then(function (key) {
-                    resolve(key);
-
-                }).fail(function (e) {
-                    reject(e);
+                FS.exists(PUBLIC_KEY_PATH).then(function () {
+                    return FS.readFile(PUBLIC_KEY_PATH)
+                    .then(function (key) {
+                        resolve(key);
+                    }).fail(function (e) {
+                        console.log('readFile error: ' + e);
+                        reject(e);
+                    });
+                }, function () {
+                    console.log('\'' + PUBLIC_KEY_PATH + '\' is not exists.');
```

Comment:
It's some hard to fix a man's habit. ㅠ.ㅠ
'exist' is a verb. So, 'is not exists' should be changed to 'does not exist'.